io/include/pcl/io/real_sense/real_sense_device_manager.h

```diff
@@ -62,10 +66,8 @@ namespace pcl
 
       class PCL_EXPORTS RealSenseDeviceManager : boost::noncopyable
       {
-
         public:
-
-          using Ptr = std::shared_ptr<RealSenseDeviceManager>;
+  using Ptr = pcl::shared_ptr<RealSenseDeviceManager>;
```

Comment:
Go through the changes in this file and keep the indentation consistent with the remaining adjacent lines.